server/lib/forest.js

```diff
@@ -281,7 +281,7 @@ function getForestModels () {
   // Hidden in Forest
   delete m.Collective.associations.Activities;
   delete m.Collective.associations.Notifications;
-  delete m.Collective.associations.ConnectedAccounts;
+  // delete m.Collective.associations.ConnectedAccounts; // xdamman: this is breaking the `allCollectives` graphql query: error: ConnectedAccount is not associated to Collective!
```

Comment:
@znarf can you look into this?

Reply:
@xdamman we're doing a cloneDeep on top of the function, so why a change here would impact the application?